compiler/src/test/java/dagger/internal/codegen/GraphValidationTest.java

```diff
@@ -2072,6 +2072,143 @@ public void subcomponentBindingConflictsWithParent() {
   }
 
   @Test
+  public void subcomponentBindingConflictsWithParentWhenSubcomponentPerformsMembersInjection() {
```

Comment:
It was correct before this PR. This adds test coverage for subcomponent members injection which didn't exist previously.